tests/topotests/bgp_evpn_mh/test_evpn_mh.py

```diff
@@ -30,6 +30,9 @@
 import os
 import re
 import sys
+import subprocess
+from functools import partial
```

Comment:
nit: unused imports
(`subprocess` not used at all, `partial` imported again in line 36/39)

Reply:
removed.